how-to-ssr/public/javascripts/oc-client.js

```diff
@@ -0,0 +1,509 @@
+'use strict';
+/* eslint no-var: 'off' */
+/* eslint prefer-arrow-callback: 'off' */
+
+var oc = oc || {};
+
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module
+    define(['exports', 'jquery'], function(exports, $) {
+      if (root.oc.cmd) { exports.cmd = root.oc.cmd; }
+      factory((root.oc = exports), $, root.head, root.document, root.window);
+    });
+  } else if (typeof exports === 'object' && typeof exports.nodeName !== 'string') {
+    // Common JS
+    factory(exports, require('jquery'), root.head, root.document, root.window);
+  } else {
+    // Browser globals
+    factory((root.oc = oc), root.$, root.head, root.document, root.window);
+  }
+}(this, function (exports, $, head, $document, $window) { // jshint ignore:line
+  // public variables
+  oc.conf = oc.conf || {};
+  oc.cmd = oc.cmd || [];
+  oc.renderedComponents = oc.renderedComponents || {};
+  oc.status = oc.status || false;
```

Comment:
Let's ensure we include all of these 4 in case we want to proceed. (cmd, conf, renderedComponents, and status)